src/OrbitGl/TrackRenderHelper.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRACK_RENDER_HELPER_H_
+#define ORBIT_GL_TRACK_RENDER_HELPER_H_
+
+#include <vector>
+
+#include "Batcher.h"
+#include "CoreMath.h"
+#include "PickingManager.h"
+
+namespace orbit_gl {
+void DrawTriangleFan(Batcher& batcher, const std::vector<Vec2>& points, const Vec2& pos,
+                     const Color& color, float rotation, float z,
+                     std::shared_ptr<Pickable> pickable);
+std::vector<Vec2> GetRoundedCornerMask(float radius, uint32_t num_sides);
```

Comment:
nodiscard and const?

Reply:
Added [[nodiscard]]. What do you mean with const? Return a `const std::vector<Vec2>`?

Reply:
GetRoundedCornerMask() const;

Reply:
It's a free function, it can't be `const`.